kubernetes/Dockerfile.mysql

```diff
@@ -3,5 +3,5 @@ FROM docker.bintray.io/jfrog/artifactory-pro:5.2.1
 MAINTAINER eldada@jfrog.com
 
 # Download the DB driver into Tomcat's lib
-RUN curl -L -o /opt/jfrog/artifactory/tomcat/lib/mysql-connector-java-5.2.10.jar https://jcenter.bintray.com/mysql/mysql-connector-java/5.2.10/mysql-connector-java-5.2.10.jar
+RUN curl -L -o /opt/jfrog/artifactory/tomcat/lib/mysql-connector-java-5.2.10.jar https://jcenter.bintray.com/mysql/mysql-connector-java/5.1.41/mysql-connector-java-5.1.41.jar
```

Comment:
The output should also be with the same name (-o /opt/jfrog/artifactory/tomcat/lib/mysql-connector-java-5.1.41.jar).
Fix and commit.